src/components/shop/balance-ui.js

```diff
@@ -6,16 +6,10 @@
  * @license Apache-2.0
  */
 
-import { getXPos, getYPos, getScaleFactor, getSafeArea } from "./shop-layout.js";
+import { getXPos, getYPos, getScaleFactor, getSafeArea, textStyle } from "./shop-layout.js";
 import { getMetrics } from "../../core/scaler.js";
 import { collections } from "../../core/collections.js";
 
-const styleDefaults = {
-    fontFamily: "ReithSans",
-    fontSize: "24px",
-    resolution: 4,
-};
-
```

Comment:
I always thought these defaults were here, instead of in the config, so that the screen had a fallback, if nothing were provided in config?

Reply:
We've now pushed the defaults into config, since lots of other stuff depends on there being defaults in there already.

Reply:
if we think it's best to have some hard coded defaults these should probably go live next to the new textStyle fn in shop-layout.js- what do you think?

Reply:
im a bit confused, is this styling all text in the shop? then other things in the config can override it?

Reply:
8d67f8b

Reply:
So it being in `balance-ui.js` was a bit of leftover baggage basically. Some stuff had defaults, some stuff didn't, bit of a mess etc. 

The state of play now is that everything uses the same default from config (if set) and then any other text elem spec'd in config can have its own `styles` object which merges with the defaults.

Reply:
ah ok, I guess its good then, as long as the config is clear to read and its easy to see what each thing should be styled with - without having to work out what gets overridden or w/e